mbed_greentea/mbed_greentea_cli.py

```diff
@@ -401,9 +401,12 @@ def run_test_thread(test_result_queue, test_queue, opts, mut, build, build_path,
                                          verbose=verbose)
 
         # Some error in htrun, abort test execution
-        if host_test_result < 0:
+        if isinstance(host_test_result, int):
```

Comment:
Well, you will get print from failing run_command inside `run_host_test()` but Iwill add extra print to signalize where we jumped outside thread loop.